ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml

```diff
@@ -86,8 +86,23 @@ tests:
   optional: true
   steps:
     test:
-    - ref: stackrox-scanner
-  timeout: 2h30m0s
+    - as: e2e-tests
+      commands: .openshift-ci/dispatch.sh e2e-tests
+      credentials:
+      - mount_path: /tmp/secret/stackrox-scanner-secrets
+        name: stackrox-scanner-secrets
+        namespace: test-credentials
+      dependencies:
+      - env: SCANNER_IMAGE
+        name: scanner
+      - env: SCANNER_DB_IMAGE
+        name: scanner-db
```

Comment:
Whatabout adding this to the stackrox-scanner step? or did you try that already?

Reply:
haven't tried it yet. Also, this is only needed for certain tests, not all. I'll need to look into utilizing the step-registry more as time goes on, but for now I just want these tests to pass